src/redux/motorcycles/api.js

```diff
@@ -0,0 +1,46 @@
+import Axios from 'axios';
+import {
+  fetchMotorcycles,
+  createMotorcycle,
+  deleteMotorcycle,
+  fetchSingleMotorcycle,
+} from './motorcycles';
+
```

Comment:
I suggest you also delete this file `(redux/motorcycles/api.js)` since all API endpoints can be imported as `API` from `('redux/api.js')`